Moodle4.5 upgrade

mirleu opened the pull request:
no functional changes

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1209227878068113

mirleu committed:
Moodle4.5 upgrade

ctam reviewed (commented):
Looks fine to me.  Note that the release variable means the release version of this plugin, not the supported Moodle version. If we want to start versioning our releases for this plugin, we should also consider tagging it with v4.5.0, v4.5.1, etc.

Also, I noticed there's a draft PR, #23, pending. What is the status of that? Any chance we can bundle that into this release?

stopfstedt commented:
agreed, let's sit on this until the test coverage is in, and the additional data points have been added (happening this week, i'm half-way there already).

stopfstedt committed:
explicitly call the setup() method on the parent class of our api test case.

we're getting dinged for not doing so by the codesniffer in a Moodle 4.5
context, see moodlehq/moodle-cs issue 106 for details.

mirleu committed:
Merge pull request #1 from stopfstedt/phpunit-setup

explicitly call the setup() method on the parent class of our api test case.

mirleu commented:
@ctam PR, https://github.com/ucsf-education/tool_ucsfsomapi/pull/23, test coverage code is merged.

ctam reviewed (approved):
lgtm
